### ISSUE TITLE
"No information" when creating events

### ISSUE THREAD

**Issue by jobisoft** [jobisoft/EAS-4-TbSync#5]
_From @GameMonkey on October 17, 2018 11:41_

## Your environment

TbSync version: 0.7.16
Thunderbird version: 6.2.1

## Expected behavior
When I want to create an event I expect to see when I, and others, are available. 

## Actual behavior
When I want to create a new event my information in the calendar is no available. 
Thunderbird directly states that my information is not available even though the calendar is synced.
![no_information](https://user-images.githubusercontent.com/7374161/47083369-a6876180-d210-11e8-959d-19be78add090.jpg)

Is this a bug? And if it is not, can it then be fixed via a feature request? 

If more information is needed I will be happy to provide.

Thanks in advance

_Copied from original issue: jobisoft/TbSync#141_

**Comment by jobisoft** [comment]
I do not think this is supported by active sync 14.0. All this advanced meeting stuff is currently not on my agenda, as I focus on bringing back EWS.

**Comment by brtptrs** [comment]
This is the only big issue we have with TbSync. It would be great if this problem could be solved.

**Comment by otto2704** [comment]
Well, this is one of the main reasons we can not move to tb (from ol) yet :-(
All the other stuff is really working awesome. 
As we are using open source wherever possible, this would be REALLY great to work (we are using zarafa/z-push as server)....

**Comment by jobisoft** [comment]
If this is not fixed by someone else, do not expect this to land before Q4 2019. This is a horribly underfunded one-man-project.

All this meeting stuff requires to implement a custom calendar class, which has direct syncing capabilities. Currently I use a standard offline storage.

**Comment by wit12345** [comment]
Hi, 
I've the same problem. Any update about this report?

Thank you.

**Comment by lciti** [comment]
Hi John

Thanks for your great work!
I have started using TBSync after [ExchangeCalendar](https://github.com/ExchangeCalendar/exchangecalendar) stopped working in TB 68. I expect many more people will do the same switch.
Everything works like a charm for me except this issue. Do you have a bounty system so that people can contribute to fixing specific issues?

Luca

**Comment by jobisoft** [comment]
This is done, since v5.2.0 (#189).

Thunderbird's attendee tab now shows real availability for an EAS account, so you can see when you and the people you are inviting are busy while picking a time.

Location lookup, which came up later in this thread, is not part of it - ActiveSync has nothing to offer there.

Out now in v5.2.3. If Thunderbird has not offered you the update yet, you find the new versions here: [TbSync](https://github.com/jobisoft/TbSync/releases/tag/v5.2.3), [EAS](https://github.com/jobisoft/EAS-4-TbSync/releases/tag/v5.2.3), [Google](https://github.com/jobisoft/google-4-tbsync/releases/tag/v5.2.3). Update all three together. Accounts set up before v5.2.2 have to be disconnected and connected again before they sync at all - back up pending data first, disconnecting deletes the account's local calendars and address books.
